src/pcap/api_test.go

```diff
@@ -306,7 +306,7 @@ func TestCapture(t *testing.T) {
 				t.Errorf("capture() unexpected error during api creation: %v", err)
 			}
 
-			var connectToTargetFn = func(ctx context.Context, req *CaptureOptions, target AgentEndpoint, creds credentials.TransportCredentials, log *zap.Logger) (captureStream, error) {
+			var connectToTargetFn = func(ctx context.Context, req *CaptureOptions, target AgentEndpoint, creds credentials.TransportCredentials, log *zap.Logger) (captureStream, error) { //nolint:revive //keep vars even if unused for better context
 				return tt.stream, tt.err
```

Comment:
I would argue based on the type you could still figure out what is put there. The name is not strictly needed.

Reply:
For some reason original author decided to do it this way, so I'll leave it be.